playground/dockerfile/Dockerfile.AppHost/qots/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Stage 1: Build the Go program
+FROM golang:1.16 AS builder
```

Comment:
```suggestion
FROM golang:1.22 AS builder
```
1.16 is prehistoric

Reply:
(not that it matters that much with this program TBH)

Reply:
A good test case for parameter support!